.circleci/config.yml

```diff
@@ -13,12 +13,15 @@ jobs:
       - run: git submodule update --remote --force
       - run: cd wp-e2e-tests && git checkout origin/${E2E_BRANCH-master}
       - restore_cache:
-          key: v1-{{ checksum "wp-e2e-tests/package-lock.json" }}
-      - run: cd wp-e2e-tests && npm install
+          keys: 
+             - v1-node10-{{ checksum "wp-e2e-tests/package-lock.json" }}
+             - v1-node10
+             
+      - run: cd wp-e2e-tests && npm ci
       - save_cache:
-          key: v1-{{ checksum "wp-e2e-tests/package-lock.json" }}
+          key: v1-node10-{{ checksum "wp-e2e-tests/package-lock.json" }}
```

Comment:
What about multi-level `.nvmrc` and `package-lock.json` like in Calypso and the full e2e tests?

Reply:
it's there... look in `restore_cache`

Reply:
there's no .nvmrc here

Reply:
ohhhhhh i see. we'd use the nvmrc in the submodule

Reply:
Should be in the submodule as `wp-e2e-tests/.nvmrc`, right?

https://github.com/Automattic/wp-e2e-tests/blob/8a13d2be9e642829da32f19c42b0fe05f576a94d/.nvmrc

Now that I see it, would it be good idea to update the submodule?

Reply:
Right, same as package-lock

Reply:
done